src/js/containers/history/HistoryContainer.jsx

```diff
@@ -17,7 +17,7 @@ class HistoryContainer extends React.Component {
 
     render() {
        return(
-        	<HistoryPage submissionID={this.props.params.submissionID}></HistoryPage>
+        	<HistoryPage submissionID={this.props.params.submissionID} type={this.props.route.type}></HistoryPage>
```

Comment:
Why does this one have opening and closing tags instead of just adding a `/` at the end of the opening one like everywhere else?